### DISCUSSION TITLE
Consiglio su accessori compatibili con Z-Bee Duo

### DISCUSSION THREAD

**Discussion by orgnemo** [indomus/forum#749] [GENERALE - Discussioni generiche]
Salve a tutti,
ho comprato Z-Bee Duo con add-on ESP-01 e firmware ser2net, ancora non è arrivato, volevo aggiungere 4 valvole termostatiche e sensori di temperatura e umidità.
Mi sapete consigliare dei modelli compatibili con Z-Bee Duo?

Grazie

**Comment by snakuzzo** [comment]
La compatibilità va vista non con Z-Bee Duo (che fa semplicemente da coordinator Zigbee), ma con Zigbee2mqtt.
[QUI](https://www.zigbee2mqtt.io/information/supported_devices.html) puoi trovare la lista dei device attualmente supportati...ma considera puoi far supportare anche device attualmente non supportati seguendo [QUESTA GUIDA](https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html)

**Reply by orgnemo** [reply]
Grazie, ho finito da poco di installare Zigbee2mqtt prima non avevo capito l'architettura, comunque ho visto questi aquara qualcuno ha delle esperienze su questi modelli?

**Reply by snakuzzo** [reply]
1. Non mettere link a shop...rimuovilo per cortesia 
2. Si chiamano Aqara
3. Credo siano i sensori temperatura/umidità Zigbee più utilizzati al mondo 😅

**Reply by KC-inDomus** [reply]
Sono questi: https://indomus.it/componenti/xiaomi-mijia-aqara-sensore-di-temperatura-di-umidita-e-pressione/
Per favore non mettere link. grazie

**Reply by orgnemo** [reply]
Scusate per il link
